metadata/eo3_deafrica.odc-type.yaml

```diff
@@ -0,0 +1,89 @@
+name: eo3
+description: EO for DE Africa
+dataset:
+  id: ['id']                               # has to be this value, can't be changed
+  sources: ['lineage', 'source_datasets']  # has to be this value, can't be changed
+  grid_spatial: ['grid_spatial', 'projection']
+  measurements: ['measurements']
+  creation_dt: ['properties', 'odc:processing_datetime']
+  label: ['properties', 'odc:label']   # not populated currently
+  format: ['properties', 'odc:file_format']  # note add to ARD metadata, only L1 currently
+
+  search_fields:
+    platform:
+      description: Platform code
+      offset: ['properties', 'eo:platform']
+
+    instrument:
+      description: Instrument name
+      offset: ['properties', 'eo:instrument']
+
+    product_family:
+      description: Product family code
+      offset: ['properties', 'odc:product_family']
+
+    region_code:
+      description: >
+        Spatial reference code from the provider.
+        For Landsat region_code is a scene path row '{:03d}{:03d}.format(path,row)'.
+        For Sentinel it is MGRS code.
+        In general it is a unique string identifier that datasets covering roughly
+        the same spatial region share.
+
+      offset: ['properties', 'odc:region_code']
+
+    cloud_cover:
+      description: Cloud cover percentage [0, 100]
+      type: double
+      offset: ['properties', 'eo:cloud_cover']
+
+    time:
+      description: Acquisition time range
+      type: datetime-range
+      min_offset:
+        - ['properties', 'dtr:start_datetime']
+        - ['properties', 'datetime']
+      max_offset:
+        - ['properties', 'dtr:end_datetime']
+        - ['properties', 'datetime']
+
+    lon:
+      description: Longitude range
+      type: double-range
+      min_offset:
+        - ['extent', 'lon', 'begin']
+      max_offset:
+        - ['extent', 'lon', 'end']
+
+    lat:
+      description: Latitude range
+      type: double-range
+      min_offset:
+        - ['extent', 'lat', 'begin']
+      max_offset:
+        - ['extent', 'lat', 'end']
+
+    # Custom DE Africa Fields
+    # Generic
+    projection:
+      description: Platform code
+      offset: ['properties', 'proj:epsg']
+
+    # Sentinel-1
+    sat_orbit_state:
+      description: The satellite orbit state, either ascending or descending
+      offset: ['properties', 'sat:orbit_state:']
+    
+    sat_relative_orbit:
+      description: The satellite relative orbit
+      offset: ['properties', 'sat:relative_orbit:']
+      type: double
```

Comment:
does this mean it's a number? I wander if this field has value starting with 0 (e.g. 086). If so, this is probably easier to handle as a string?

Reply:
I'm actually not sure without looking at the STAC files.

Reply:
Numbers were like `123`. We can handle as a string though, yep.